activesupport/lib/active_support/core_ext/object/blank.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+
+require_relative "../regexp"
```

Comment:
Is this fine needed? There is no regexp file in the object folder.

Reply:
This line seems to have come from this commit: https://github.com/gwincr11/rails/commit/8da30ad6be34339124ba4cb4e36aea260dda12bc

Reply:
https://github.com/gwincr11/rails/commit/8da30ad6be34339124ba4cb4e36aea260dda12bc#diff-eab83e40fb6006545eae52b2a4ceac08

Reply:
The line 6 is doing the same thing. You can remove it